### ISSUE TITLE
About v-WERP

### ISSUE THREAD

**Issue by DebbieSimtth** [JulioSoteloParraguez/4D-Flow-Matlab-Toolbox#1]
Dear Dr. Julio Sotelo,
 I wonder if you have the "vWERP/" folder that go inside the 4D flow APP folder? I contacted David Marlevi,but I didn't get responsd. Could you share it with me if you got it? Thanks a lot.

**Comment by JulioSoteloParraguez** [comment]
Hi Debbie, could you send me an email to julio.sotelo@uv.cl, and I'll try to communicate with David Marlevi through my email. Unfortunately I can't share that folder because is not part of my research.  Best regard!

**Comment by DebbieSimtth** [comment]
> Hi Debbie, could you send me an email to [julio.sotelo@uv.cl](mailto:julio.sotelo@uv.cl), and I'll try to communicate with David Marlevi through my email. Unfortunately I can't share that folder because is not part of my research. Best regard!

Thank you so much Dr. Julio Sotelo, I already send you an email. I do appreciate you getting back to me.

**Comment by DebbieSimtth** [comment]
Dear Dr. Julio Sotelo, I'm so sorry I came here to see you again.Please forgive me for interrupting.I have sent you an email from my institutional email(anyangu@stu.gzhmu.edu.cn)  and Google email(guanyan1996@gmail.com), but I haven't received your reply yet.I don't know if they have been filtered into junk mailboxes again, or whether you are too busy to find time to check.I will benefit a lot if I can get your help.Thank you so much.Best wish.

**Comment by Xtaltec** [comment]
Hello,

I am also very interested in calculating the relative pressure through v-WERP. Did you manage to contact David in the end?

Kind regards,
Xabier

**Comment by DebbieSimtth** [comment]
Yeah. But I didn 't study v-WERP in the end. Sorry, I can't help you.

Best regard,
Debbie



At 2023-03-17 02:55:29, "Xabier Morales Ferez" ***@***.***> wrote:

Hello,

I am also very interested in calculating the relative pressure through v-WERP. Did you manage to contact David in the end?

Kind regards,
Xabier

**Comment by JulioSoteloParraguez** [comment]
Dear all 

You can write to David at this email david.marlevi@ki.se. 
Unfortunately I cannot share the vWERP/ code, I recommend that you write to David from an institutional email, because is a more formal request.

Best

**Comment by DebbieSimtth** [comment]
Dear Xabier,
Thank you very much for your advice.But I don't need the method about v-WERP now.


Best wish,
Debbie

















At 2023-03-20 20:49:58, "Julio Sotelo" ***@***.***> wrote:

Dear all

You can write to David at this email ***@***.***
Unfortunately I cannot share the vWERP/ code, I recommend that you write to David from an institutional email, because is a more formal request.

Best

**Comment by Xtaltec** [comment]
Hi, 

Thank you both very much for the rapid response. I have managed to contact David through the email shared by Julio. 

Kind regards,
Xabier
